### ISSUE TITLE
Support .ods as well as .xlsx files

### ISSUE THREAD

**Issue by coolshaurya** [nushell/nushell#1106]
**Is your feature request related to a problem? Please describe.**
Currently nu supports `.xlsx` files which is a proprietary MS file format. This being an open source project, support for the OpenDocument `.ods` files should also be there.

**Describe the solution you'd like**
doing `open xyz.xlsx` gives us -

```
━━━━━━━━━━━━━━━━━
 Sheet1 
─────────────────
 [table 26 rows] 
━━━━━━━━━━━━━━━━━
```
Similarily doing `open xyz.ods` should give us the same output.

**Describe alternatives you've considered**
none

**Additional context**
Currently, doing `open xyz.ods` gives a bunch of binary output -
```
❯ open xyz.ods
Length: 4498 (0x1192) bytes
0000:   50 4b 03 04  14 00 00 00  00 00 00 00  00 00 85 6c   PK.............l
0010:   39 8a 2e 00  00 00 2e 00  00 00 08 00  00 00 6d 69   9.............mi

....

1170:   6d 61 6e 69  66 65 73 74  2e 78 6d 6c  50 4b 05 06   manifest.xmlPK..
1180:   00 00 00 00  06 00 06 00  5a 01 00 00  22 10 00 00   ........Z..."...
1190:   00 00                                                ..
```

**Comment by jonathandturner** [comment]
Because xlsx and ods support use the same dependencies, you should be able to copy the `from_xlsx` command and make a `from_ods` command, changing the name and description to match

**Comment by coolshaurya** [comment]
Working on it - https://github.com/coolshaurya/nushell/tree/command-from-ods

**Comment by coolshaurya** [comment]
closing issue as fixed by PR #1161
